docs/DeveloperGuide.md

```diff
@@ -315,36 +327,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2b3. Chef clicks the "file fixed" button
 
       Use case resumes from step 2.
-
-
-
+    
 #### **Use case: Add a recipe**
 
-
 **MSS**
 
 1.  Chef requests to add a recipe
-2.  Book prompts the user for the name, ingredients and steps for the recipe
-3.  Chef keys in details for each section one at a time
+2.  Book prompts the user for the name, duration, portion, tags, ingredients and steps for the recipe
+3.  Chef keys in details for each section in a single input, following the specified format
 4.  Book adds the recipe to a database
 
 **Extensions**
 
-* 3a. The name, ingredients or steps typed by Cook are empty.
+* 3a. The name inputted by chef is missing
     * 3a1. Book shows an error message.
-    * 3a2. Book requests for the correct entry or for the Cook to type "/exit" if he/she chooses not to continue.
+    * 3a2. Book requests for the name of the recipe to be keyed in
 
       Use case resumes from step 3.
 
-* 4a. The given recipe is a duplicate. The name, ingredients and steps coincide with another recipe
+* 3b. The fields inputted by chef is in the wrong format
+    * 3b1. Book shows an error message.
+    * 3b2. Book requests for the fields to be input in an appropriate format
+  
+      Use case resumes from step 3.
+
+* 4a. The given recipe is a duplicate. The name, duration, portion, tags, ingredients and steps coincide with another recipe
     * 4a1. Book shows a message that states there already exist such recipe.
     * 4a2. The recipe keyed is not added.
 
       Use case ends.
+    
+#### **Use case: View a recipe**
 
+**MSS**
 
-#### **Use case: Delete a recipe**
+1. Chef requests to list recipes
+2. Book shows a list of ***all*** recipes
+3. Chef requests to view a specific recipe in the list
+4. Book return the specified recipe
```

Comment:
(i'm assuming `view` refers to the double-click to open the recipe in a new window)
you don't actually have to run `list` first before viewing a recipe. it can be done from a filter view, such as after a `find` command is run